src/main/java/Repository/ApartmentStorage.java

```diff
@@ -0,0 +1,61 @@
+package Repository;
```

Comment:
in general - package names in uppercase
just repository 
the same for other packages